test/parseExpectationsTest.js

```diff
@@ -6,12 +6,12 @@ const h = require("fido2-helpers");
 const coerceToArrayBuffer = h.functions.coerceToArrayBuffer;
 const abEqual = h.functions.abEqual;
 
-describe("parseExpectations", function() {
-	it("parser is object", function() {
+describe("parseExpectations", function () {
```

Comment:
Not sure if I understand this formatting change - lets keep it the old way

Reply:
Ok, I have added a rule to eslint configuration to make this mandatory.